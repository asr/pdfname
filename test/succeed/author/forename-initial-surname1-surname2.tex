\documentclass{article}

\pdfinfo{%
  /Author (Forename M. Surname1-Surname2)%
  /Title  (Title)%
  /CreationDate (D:2017)%
}

\begin{document}
Hello World!
\end{document}

