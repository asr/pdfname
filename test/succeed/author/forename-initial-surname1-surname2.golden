ret > ExitSuccess
out > The full path name will be /tmp/surname1-surname2-2017.title.pdf
out >
